code/_helpers/global_lists_vr.dm

```diff
@@ -56,7 +63,30 @@ var/global/list/digestion_sounds = list(
 		'sound/vore/digest11.ogg',
 		'sound/vore/digest12.ogg')
 
+var/global/list/digest_none = list(null)
```

Comment:
Won't this result in  `sound(null)` being sent to the client?  Have you tested what happens when you do this?

Reply:
(Hint: Sending 'null' as a sound stops all sounds playing on the client)